Lib/idlelib/autocomplete.py

```diff
@@ -225,4 +225,4 @@ def get_entity(self, name):
 
 if __name__ == '__main__':
     from unittest import main
-    main('idlelib.idle_test.test_autocomplete', verbosity=2)
+    main('test.test_idle.test_autocomplete', verbosity=2)
```

Comment:
Actually, if tests were not installed, this would just crash, and prevent running any htest that follows.  Running a file's unittest would have to be made conditional (or the exception caught).  This if tests were moved, this should be encapsulated in a util function that would fail gracefully.